### PR TITLE
RLS on Windows

### PR DESCRIPTION
**Pull request by Et7f3**
This PR fix 2 bugs on windows and it is related to https://github.com/jaredly/reason-language-server/pull/199
It is enough for me to have https://atom.io/packages/ide-reason working on my computer

### CODE REVIEW

**Comment by Et7f3** [comment]
I haven't fixed OCAMLLIB variable bug (I have set a global one as workaround) this is resolve in #199
and I don't understand why util/Commands.re change in #199 (maybe a nother bug I haven't seen) and  didn't touch to test (maybe add one or fix them if they are present)

**Comment by Et7f3** [comment]
Where is absify used ? Maybe there is  other pieces of useless code.

**Comment by zetashift** [comment]
@Et7f3 you mean you set a global PATH variable as a workaround for the OCAMLLIB error?

File: src/analyze/BuildSystem.re [hunk 1]
```diff
@@ -264,10 +264,24 @@ let isRunningInEsyNamedSandbox = () => {
...
+  let ret = if (isRunningInEsyNamedSandbox()) {
     getLine("which " ++ exeName, ~pwd)
   } else {
     getLine("esy which " ++ exeName, ~pwd)
+  };
+  if (Sys.win32) {
+    switch ret {
+      | RResult.Ok(ret) =>
+        let ret = if (isRunningInEsyNamedSandbox()) {
+          getLine("cygpath -w " ++ ret, ~pwd)
```
Comment (jaredly) [inline src/analyze/BuildSystem.re]: this is only if you're running under cygwin, right? Which I don't think is necessarily the case?

Comment (Et7f3) [reply src/analyze/BuildSystem.re]: We are on windows `if (Sys.win32)` and esy on windows have cygwin with esy-bash: so I assume it is necessarily the case. Maybe ping a maintener of esy :shrug:

Comment (Et7f3) [reply src/analyze/BuildSystem.re]: I can add a try with.

**Comment by Et7f3** [comment]
@zetashift No You can set env variable and I have done this 
image
if you are on Windows: I open Windos menu type `env`
image

Should be change environnement variable or something else in english

image

then you can add key/pair value and set manually OCAMLLIB

**Comment by jaredly** [comment]
thanks!

**Pull request merged**